src/AppRouter.js

```diff
@@ -17,37 +20,34 @@ const AppRouter = () => (
       path="/:topic"
       render={({ history, match }) => {
         if (topicKeys.includes(match.params.topic)) {
+          const activeTopic = topics.find(t => t.key === match.params.topic);
+          const { elements } = activeTopic;
```

Comment:
Can't we extract `elements` from `topics` inside TrafimageMaps?

Reply:
Not exactly sure what you mean.

Reply:
To me it looks like this can be moved into the TrafimageMaps component, but maybe I'm missing something.

Reply:
yes and that's break menu display.

Reply:
Menus display fixed